src/main/java/com/nha/abdm/wrapper/hip/hrp/GatewayCallbackController.java

```diff
@@ -260,4 +213,35 @@ private ResponseEntity<GatewayCallbackResponse> handleIllegalDataStateException(
             .build(),
         HttpStatus.INTERNAL_SERVER_ERROR);
   }
+
+  /**
+   * Callback from ABDM gateway of consent to dataTransfer
+   *
+   * @param hipNotifyRequest Has careContexts details for dataTransfer
+   */
+  @PostMapping({"/v0.5/consents/hip/notify"})
+  public ResponseEntity<GatewayCallbackResponse> initiateDataOnNotify(@RequestBody HIPNotifyRequest hipNotifyRequest)
```

Comment:
lets keep the names of methods consistent like you have initiateDataOnNotify as method name and workflowmanager's method as initiateConsentOnNotify. This should not be "ON" notify. lets reduce the confusion. if the url says hip notify lets keep the method name very similar to that like `hipNotify` and same for workflowmanager's method `workflowManager.hipNofify`